### ISSUE TITLE
unable to `pack`: "permission denied when trying to kill process 1"

### ISSUE THREAD

**Issue by tomholford** [urbit/vere#198]
I have a script that runs `pack` and `meld` on a pier, e.g. `./pack-and-meld datder-sonnet/`
```bash
#!/usr/bin/env bash

echo "Ensure your ship is not running."

read -p "Press enter to continue"

./urbit pack $1 && ./urbit meld $1
```

However, it is throwing this error which I have not seen before:
```
disk: permission denied when trying to kill process 1!
Terminated
```

Looking at `htop`, process 1 is `init`, not sure why it's trying to kill that?
<img width="612" alt="image" src="https://user-images.githubusercontent.com/16504501/216711799-5b1e2b13-201b-44b8-a84c-3d9472082951.png">

For context, versions:
```
$ ./urbit --version
urbit 1.18
gmp: 6.2.1
sigsegv: 2.14
openssl: OpenSSL 1.1.1q  5 Jul 2022
libuv: 1.44.2
libh2o: 0.13.6
lmdb: 0.9.29
curl: 7.85.0
```

And system:
```
$ uname -a
Linux datder-sonnet 5.15.0-58-generic #64-Ubuntu SMP Thu Jan 5 11:43:13 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by zalberico** [comment]
I've see this - usually means your ship is running
- Are you trying to run pack with the binary and your ship is also running?
- Is it running under a root user and you're trying to run pack as a different user?

**Comment by tomholford** [comment]
Yeah false alarm, I think I did not shut it down! Sleepy dev ops is a bad idea 😴  Closed as working as intended

**Comment by tomholford** [comment]
@zalberico So, I encountered this again. Here's a screenshot showing the output of `docker ps` which indicates that the `urbit` container is not running:

<img width="513" alt="image" src="https://user-images.githubusercontent.com/16504501/217405377-cb673333-2ac0-4929-a5d2-47be56331f98.png">

For comparison, here is the output for a moon that is running in a container:

<img width="712" alt="image" src="https://user-images.githubusercontent.com/16504501/217405466-0eb4677c-3ab8-4b35-962e-fd8aec000e89.png">

This happened after I used the `docker-compose down` command to shut down the container. Perhaps it is not shutting down the `vere` process gracefully?
